starter-code/routes/gif.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+/* GET home page. */
+router.get('/', (req, res, next) => {
+  if (req.session.currentUser) {
+    return res.redirect('gif')
```

Comment:
Are you redirecting to the same path you are? Furthermore, without '/' will go to gif/gif.